### PR TITLE
Ampi privatisation tutorial

### PR DESCRIPTION
**Pull request by raj-aayush**

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Why was the previous pull request with this patch closed?

**Comment by raj-aayush** [comment]
> Why was the previous pull request with this patch closed?

The Makefile in the MPI version of the PENNANT program created a slight issue with the executable. I didn't notice that when I first created the pull request. It's fixed in this version now.

File: examples/ampi/privatisation_tutorial/nm_globals.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```
Comment (matthiasdiener) [inline examples/ampi/privatisation_tutorial/nm_globals.sh]: Is this copy of nm_globals.sh required?

Comment (raj-aayush) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: This copy is placed there for convenience. I could get rid of it if needed.

Comment (matthiasdiener) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: Would it make sense to replace the copy with a symlink or so?

Comment (raj-aayush) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: That's a good point. I will make that change. So will this pull request need to closed?

Comment (matthiasdiener) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: No need to close this pull request, you can just push another commit to the branch to update it.

Comment (raj-aayush) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: Made the changes to nm_globals.sh. Please verify.

Comment (matthiasdiener) [reply examples/ampi/privatisation_tutorial/nm_globals.sh]: looks good to me

**Comment by evan-charmworks** [comment]
It looks like some other files too. Maybe `make clean` should be run.

**Comment by evan-charmworks** [comment]
This is good work, but I don't think we should bring external MPI codes into the main Charm repository.

**Comment by ericjbohm** [comment]
We already have a few, such as the mpich tests.  So its more of a question
of quantity and what utility they bring.

**Pull request closed without merging**